README.md

```diff
@@ -25,154 +22,334 @@
+- Biodiversity value (quan)
+- Biodiverse areas at risk (qual)
+- Areas providing biodiversity opportunities (qual)
+
+### 2.1.2 Research question 2: Which areas are the most in need for flood protection?
+- Flood risk (qual)
+- Surface temperature (quan)
+- Vegetation health (qual)
+- 
+### 2.1.3 Research question 3: Where can improvements in quality of life balance stakeholder interests in areas around the stream?
+- Public amenities (quan)
+- Accessibility of stream (qual)
+- Physical comfort level (quan)
 
 ### 2.1.3 Envisioned uses of the dataset
-- ...
-- ...
+- Evaluate the environmental attributes around the stream and distinguish the habitat quality grades
+- Support climate-adaptive stream design by identifying high flood-risk areas
+- Evaluate the accessibility of each section of the stream, as well as the comfort and convenience for people to use
 
 ## 2.2 Methods for processing the data
-- ...
-- ...
+- MCDA to select the influencing factors according to the research questions and assign weights for them in order to make the final decision
+- Fuzzy AHP to determine which data has a higher weight
+- K means to construct typology of data
 
 ## 2.3 Instrument- or software-specific information
-- Software version X.X was used for ...
+- QGIS version 3.34.12 was used for making maps
+- R studio version 4.5.0 was used for editing reports and gathering products
+
 
 # 3. FILE OVERVIEW
-Are there multiple versions of the dataset? Yes/No
+There are two versions of this dataset.
 
 ## 3.1 File List
 
-### 3.1.1 Data category A
-- "filename.extension": brief description of file
-
-### 3.1.2 Data category B
-- "filename.extension": brief description of file
-
-### 3.1.3 Data category C
-- "filename.extension": brief description of file
-
-### 3.1.4 Data category D
-- "filename.extension": brief description of file
+### 3.1.1 Urban Stream Restoration: Bílá Nisa Data
+- "Bila_nisa_urban_stream_project.csv" and "Bila_nisa_urban_stream_project.gpgk": Normalized Data on Biodiversity, Climate Adaptation & Quality of Life Along The Bílá Nisa Stream in Jablonec nad Nisou
 
 ## 3.2 Relationship between files:
-The following tables (csv files) employ a foreign key to refer to the primary key (unique identifier) in one or more other table(s):
-
-"filename.extension"
-- abc used as foreign key to "otherfile.extension"
+N/A
 
 ## 3.3 File formats and naming conventions
 ### 3.3.1 File formats
-- extension - type of data
+Excel file: .CSV dataset
+Geopackage file: .GPGK
 
 ### 3.3.2 Naming conventions
-- files named lower case, spaces replaced with dashes (dash-case)
-- in tabular data, variable names snake case
+N/A
 
 # 4. DATA-SPECIFIC INFORMATION
 
-- Missing data code: NA
+- Missing data code: N/A
 - Not Applicable: N/A
 
-## 4.1 Data Category A
+## 4.1 Urban Stream Restoration: Bílá Nisa Data
 
-### 4.1.1 filename.extension
-1. Number of variables: 
+### 4.1.1 UrbanStreamRestoration_Bila_Nisa.csv
```

Comment:
If you publish the gpkg version too, make sure you add that second file name here too. No need to repeat the content, as it is the same.